core/input/input.cpp

```diff
@@ -713,6 +722,7 @@ void Input::_parse_input_event_impl(const Ref<InputEvent> &p_event, bool p_is_em
 						action.released_process_frame = Engine::get_singleton()->get_process_frames();
 					}
 					action.pressed = MAX(action.pressed - 1, 0);
+					action.axis_pressed = false;
```

Comment:
I believe, that it is possible to fix comment https://github.com/godotengine/godot/pull/81170#issuecomment-1699867615 with:
``` c++
					if (jm.is_valid()) {
						if (action.axis_pressed) {
							if (action.pressed == 1) {
								action.released_physics_frame = Engine::get_singleton()->get_physics_frames();
								action.released_process_frame = Engine::get_singleton()->get_process_frames();
							}
							action.pressed = MAX(action.pressed - 1, 0);
							print_line("p3=", action.pressed);
							action.axis_pressed = false;
						}
					} else {
						action.pressed--;
					}
```

Reply:
Found a simpler solution.